pages/sushibar.html

```diff
@@ -298,4 +298,4 @@ <h3 class="card-title">Who's serving?</h3>
     });
 </script>
 
-</html>
\ No newline at end of file
+</html>

```

Comment:
I am not sure why this edit is here... perhaps it is because I used the github.com editor to make the above changes